tasks/prime.py

```diff
@@ -272,7 +274,80 @@ def on_start(self):
             name="/internal/service_members/{serviceMemberId}/backup_contacts",
             data=json.dumps(payload),
             headers={"content-type": "application/json"},
-            **self.user.cert_kwargs,
+        )
+
+        # Setup customer order
+        overrides = {
+            "service_member_id": service_member_id,
+            "issue_date": datetime.now().strftime("%Y-%m-%d"),
+            "report_by_date": datetime.now().strftime("%Y-%m-%d"),
+            "orders_type": "PERMANENT_CHANGE_OF_STATION",
+            "orders_type_detail": "HHG_PERMITTED",
```

Comment:
I don't think this field gets a value during customer onboarding, only when the office user updates the orders later.

Reply:
Nice catch. Yeah it doesn't look like the customer flow sends this